client/src/components/broadbandCard/BroadbandCard.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import Button from 'material-ui/Button';
+
+class BroadbandCard extends Component {
```

Comment:
Esse cara pode virar um [_Function Component_](https://reactjs.org/docs/components-and-props.html), porque ele não tem estado e nem usa _hooks_.

```js
function BroadbandCard({ name, totalPrice }) {
  return (
    <div>
      <p>{Array.isArray(name) ? name.join(' + ') : name}</p>
      <p>{totalPrice}</p>
      <Button variant="raised" color="primary">
        Comprar
      </Button>
    </div>
  );
}
```

Reply:
Feito